.github/workflows/ci.yml

```diff
@@ -47,6 +47,8 @@ jobs:
           profile: minimal
           override: true
 
+      - run: rustup target add thumbv7m-none-eabi
```

Comment:
do we need to test against this target? I would be happy enough to have CI just check that no-std builds, against a default target.

Reply:
`no_std` crates can accidentally link to std by accident, in some weird cases. :face_with_spiral_eyes: To catch these errors, you can build on a target that doesn't have a libstd.

Reply:
okay cool, I hadn't heard about that trick before. :)